lib/schedule/data.ex

```diff
@@ -271,6 +271,13 @@ defmodule Schedule.Data do
     |> RoutePattern.by_direction()
   end
 
+  @spec feed_info(binary()) :: FeedInfo.t()
+  defp feed_info(feed_info_data) do
```

Comment:
I've done a couple of these where the file parsing is in the struct's file with `from_csv_row`. E.g. https://github.com/mbta/skate/pull/604/files#diff-b097daeae22707cb8f51d331dc2903c1R44 and https://github.com/mbta/skate/blob/master/lib/schedule/hastus/activity.ex#L66 , and so far I'm liking it a lot more than doing the file parsing in `data.ex`. It keeps `data.ex` smaller and more focused, and now that there's a direct correspondence between `schedule/gtfs/*` files and gtfs files, it feels like it fits well in that file.